x-pack/heartbeat/monitors/browser/browser.go

```diff
@@ -35,12 +35,10 @@ func create(name string, cfg *common.Config) (p plugin.Plugin, err error) {
 		logp.Info("Synthetic browser monitor detected! Please note synthetic monitors are a beta feature!")
 	})
 
-	curUser, err := user.Current()
```

Comment:
Does it need to be here even now? If we havent set `BEAT_SETUID_AS`, do we run as heartbeat user?

Reply:
No, in that case we're running as root, and we bail with the error message. If that env var isn't set we don't know who to run as and we can't figure it out any other way. Keep in mind `heartbeat` is only the correct user in the heartbeat docker image. In the elastic-agent container the correct user is `elastic-agent`